components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2783,4 +2820,103 @@ void RewardsServiceImpl::OnDeleteActivityInfo(
   }
 }
 
+void RewardsServiceImpl::GetAutoContributeCount(
+    const rewards_counter::OnRewardsCountedCallback callback) {
+  GetReconcileStamp(
```

Comment:
so this is still not quite right. Any calls to the DB should always go through the lib so this needs to call out to ledger_-> GetAutoContributeCount
There's also no reason to make an extra db call to get the reconcile stamp. `GetAutoContributeCount` can just do a single query based on the last reconcile stamp

Reply:
This now goes through ledger